datumaro/components/converter.py

```diff
@@ -62,12 +66,17 @@ def _make_image_filename(self, item, *, name=None, subdir=None):
         subdir = subdir or ''
         return osp.join(subdir, name + self._find_image_ext(item))
 
+    def _make_pcd_filename(self, item, *, name=None, subdir=None):
+        name = name or item.id
+        subdir = subdir or ''
+        return osp.join(subdir, name + '.pcd')
```

Comment:
This function mostly replicates `_make_image_filename` - perhaps you could factor the common part out?

Reply:
Refactored